README.md

```diff
@@ -221,15 +225,25 @@ Run the following command to check it meets internal standards:
 ```shell
 spectral lint step9.yaml --verbose --ruleset rulesets/cicd.yaml --format pretty
 ```
+In return:
+
+```shell
+Found 58 rules (9 enabled)
+Linting /Users/agentle/src/DEVWKS-2525/step9.yaml
+
+0 Unique Issue(s)
+✖
+No results with a severity of 'error' found!
+```
 
```

Comment:
While screenshots are nice, it's also meaningful to copy and paste the intended output for the reader to see in text.

Reply:
integrated